.github/workflows/main.yml

```diff
@@ -69,29 +71,67 @@ jobs:
         with:
           chart-dir: ${{ env.CHART_DIR }}
 
-      - name: Get commit message
-        id: get-commit-message
-        shell: bash
-        run: |
-          MSG=$(git log --pretty=format:%s --max-count=1)
-          echo "message=$MSG" >> $GITHUB_OUTPUT
+  release:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.event_name == 'push'
+    outputs:
+      version: ${{ steps.calculate-next-internal-version.outputs.next-prerelease }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Parse next release
+        id: helm-parse-next-release
+        uses: Alfresco/alfresco-build-tools/.github/actions/helm-parse-next-release@v1.22.0
+        with:
+          chart-dir: ${{ env.CHART_DIR }}
+
+      - id: calculate-next-internal-version
+        name: Calculate next internal release
+        uses: Alfresco/alfresco-build-tools/.github/actions/calculate-next-internal-version@v1.22.0
+        with:
+          next-version: ${{ steps.helm-parse-next-release.outputs.next-release }}
 
-      - name: Publish Helm chart
-        uses: Alfresco/alfresco-build-tools/.github/actions/helm-publish-chart@v1.22.0
+      - id: helm-release-and-publish
+        name: Release and publish helm chart
+        # TODO: clean before release
+        uses: Alfresco/alfresco-build-tools/.github/actions/helm-release-and-publish@dev-igdianov-AAE-11371
         with:
-          chart-package: ${{ steps.package-helm-chart.outputs.package-file-path }}
-          helm-charts-repo: ${{ env.HELM_REPO }}
-          helm-charts-repo-branch: ${{ env.HELM_REPO_BRANCH }}
-          helm-charts-repo-subfolder: ${{ env.HELM_REPO_SUBFOLDER }}
-          helm-charts-repo-base-url: ${{ env.HELM_REPO_BASE_URL }}
-          token: ${{ secrets.BOT_GITHUB_TOKEN }}
+          version: ${{ steps.calculate-next-internal-version.outputs.next-prerelease }}
+          chart-dir: ${{ env.CHART_DIR }}
+          helm-repository: ${{ env.HELM_REPO }}
+          helm-repository-branch: ${{ env.HELM_REPO_BRANCH }}
+          helm-repository-subfolder: ${{ env.HELM_REPO_SUBFOLDER }}
+          helm-repository-base-url: ${{ env.HELM_REPO_BASE_URL }}
+          helm-repository-token: ${{ secrets.BOT_GITHUB_TOKEN }}
           git-username: ${{ secrets.BOT_GITHUB_USERNAME }}
-          git-message: ${{ steps.get-commit-message.outputs.message }}
-          do-push: ${{ github.event_name == 'push' }}
```

Comment:
Please keep that input as I added it on the helm-release-and-publish action too

Reply:
fixed